app/services/analytics.rb

```diff
@@ -130,9 +130,7 @@ def sp_request_attributes
     return if resolved_result.nil?
 
     attributes = resolved_result.to_h
-    attributes[:component_values] = resolved_result.component_values.map do |v|
-      [v.name.sub('http://idmanagement.gov/ns/assurance/', ''), true]
-    end.to_h
+    attributes[:component_values] = resolved_result.component_names
```

Comment:
Hmm, are you proposing eliminating `component_values` and adding `component_names`?

Reply:
ah, i think that does make sense

Reply:
I'm not totally sure that is needed if only because all impacted production queries (5 total) wouldn't be referencing this field any more (they should all be switched to using `sp_request.facial_match` or `sp_request.identity_proofing`), but maybe I'm not fully grokking the concern. I'll ping you offline.

Reply:
Resolved offline. (See this [thread in GitLab](https://gitlab.login.gov/lg-people/Melba/backlog-fy24/-/issues/112#note_203854) for more context)

Reply:
@zachmargolis wanted to make sure this felt resolved to you. it still feels like it makes sense to add a new data element rather than alter the old one, in case anyone is using the old value in ad-hoc queries. but i'll defer to whatever decision ya'll have come to.

Reply:
If I was writing this to optimize for querying, I would:
- keep the hash-style format (it's easier to query in Cloudwatch)

If we are set changing to using a list, then I would:
- change the key name to make it easier to query sub-fields across future + past events

Reply:
Just to clarify a particular point:  We _don't_ want to optimize for querying. The logged ACR values mean different things at different points in the identity verification and authentication request flows. The more useful information is captured in the rest of the `sp_request` attributes.

Reply:
to step back a bit, i think what we were trying to do with this change was create a consistent way for this information to be logged/queried, since the original implementation doesn't really translate to the new acr values (ie, some requests would have `ial/2 => true`, and others would have `url:acr.login.gov:verified => true`).

while the ticket i wrote originally said that we should update the hash syntax to array, zach's pointed out there is value in keeping the type of data consistent over time. so i am now thinking we should keep that value the way it was, and add a new key name for the full list of component names. i'll update the ticket to indicate as such.

Reply:
This has been updated to maintain the old sp_request.component_values behavior and adds the `sp_request.component_names` field.